### PR TITLE
Add StickK

### PR DESCRIPTION
**Pull request by kymckay**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -7940,6 +7940,17 @@
         ]
     },
 
+    {
+        "name": "StickK",
+        "difficulty": "hard",
+        "url": "https://stickk.zendesk.com/hc/en-us/requests/new",
+        "notes": "From the dropdown select \"I'd like to request to end my account\" and then fill in and submit the form.",
+        "domains": [
+            "stickk.com"
```
Comment (tupaschoal) [inline _data/sites.json:7949]:
```suggestion
            "stickk.com",
```

**Review by tupaschoal — CHANGES_REQUESTED** [review]
I fixed a JSON format issue, but it still needs to be in alphanumeric order.

**Comment by kymckay** [comment]
My bad, must have done this one in a rush. I've just squashed all the fixes into a single commit now.

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**